UKHomeOffice/sgmr-service
# Fix duplicated ID & mistyped ID

### AC / Description of changes

Issue Description
Ensures every id attribute value used in ARIA and in labels is unique
 - `aisInput` was used in two fields as ID, updated one to be `aisNumberInput`
- `mmsiInput` was used in two fields as ID, updated one to be `mmsiNumberInput`


Issue Description
Ensures every form element has a label
 - pleasure craft form: `vatPaidNo`, typo in label had `vatPaidNNo` instead of `vatPaidNo`, fixed


### To Test


- go to pleasure craft page 2
- select yes to all the options
- run axe dev tools
- > there should be no issues for duplicated id’s or missing labels


### Notes

---

**Remember to merge to feature branches not `master`.**
